README.md

```diff
@@ -4,13 +4,12 @@ Fornalder ("Bygone Age") is a small utility that can be used to ingest
 commit data from collections of git repositories and visualize it in
 various ways.
 
-It was used to generate the graphs in [this blog post](https://hpjansson.org/blag/2020/12/16/on-the-graying-of-gnome/). It's made
-to work with data going back a long time (a decade or two); with
+It was used to generate the graphs in [this blog post](https://hpjansson.org/blag/2020/12/16/on-the-graying-of-gnome/). It's made to work with data going back a long time (a decade or two); with
 shorter time spans the output will probably look quite bad.
 
 ## Building
 
-Make sure you have the Rust prerequisites installed, then:
+Fork the project repository and make sure you have the Rust prerequisites installed, then:
```

Comment:
This looks fine, but I think it should say "Clone" instead of "Fork". Forking isn't required. You could also add the command to clone it (`git clone ...`) if you like.